packages/deployments/contracts/contracts_forge/RelayerFeeRouter.t.sol

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.11;
+
+import "./ForgeHelper.sol";
+
+import "../contracts/nomad-xapps/contracts/relayer-fee-router/RelayerFeeRouter.sol";
+import {Home} from "../contracts/nomad-core/contracts/Home.sol";
+
+import "@openzeppelin/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+// running tests (with logging on failure):
+// yarn workspace @connext/nxtp-contracts test:forge -vvv
+// run a single test:
+// yarn workspace @connext/nxtp-contracts test:forge -m testAddRouterAlreadyApproved -vvv
+
+// other forge commands: yarn workspace @connext/nxtp-contracts forge <CMD>
+// see docs here: https://onbjerg.github.io/foundry-book/index.html
+
+contract MockHome {
+  function dispatch(
+    uint32 _destinationDomain,
+    bytes32 _recipientAddress,
+    bytes memory _messageBody
+  ) external {
+    1 == 1;
+  }
+}
+
+contract RelayerFeeRouterTest is ForgeHelper {
+  using TypedMemView for bytes;
+  using TypedMemView for bytes29;
+  using RelayerFeeMessage for bytes29;
+
+  // ============ Libraries ============
+  using stdStorage for StdStorage;
+
+  event Send(
+    uint32 domain,
+    address recipient,
+    uint256 amount,
+    bytes32[] _transactionIds,
```

Comment:
nit: doesn't need underscore here lol

but actually, we should put these `Send` and `Receive` events in an `interface IRelayerFeeBridge`

Reply:
If the contract size is small enough i'm actually okay with not having it an interface, but i recognize that's inconsistent throughout the repo. 

Also this is a test contract so really it should still be able to pull these events without defining them

Reply:
> Also this is a test contract so really it should still be able to pull these events without defining them
I think it isn't possible to emit an event from an external contract, nor to import events from other contracts.

Regarding having an `IRelayerFeeBridge, what would you say is the best place to put it?

Reply:
i'm not sure if this is possible unfortunately -- i believe @patitonar looked into this a bit as well